README.md

```diff
@@ -1 +1,115 @@
-# Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](Html,Css)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](Abenezer)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖  Personal Portfolio <a name="about-project"></a>
+
+**Cap stone project** is a static portfolio website.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The Capstone project is built with html, css, javascript and linter configration with git workflow.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
```

Comment:
- [ ] Kindly add JavaScript here as you have used it to load the speakers dynamically and for the mobile menu.